internal/v4/content_test.go

```diff
@@ -145,6 +138,75 @@ func (s *APISuite) TestServeDiagram(c *gc.C) {
 	})
 }
 
+func (s *APISuite) TestServeDiagramNoPosition(c *gc.C) {
```

Comment:
This test seems to overlap substantially with TestServeDiagram. Perhaps it would be better to just check that we can get the diagram for a bundle with no position info (i.e. verify that we no longer get an error) without copy and pasting this rather large and involved test?